matlab/test/arrow/array/tTime32Array.m

```diff
@@ -30,6 +30,14 @@ function Basic(tc)
             times = seconds(1:4);
             array = tc.ArrowArrayConstructorFcn(times);
             tc.verifyInstanceOf(array, "arrow.array.Time32Array");
+            tc.verifyEqual(array.toMATLAB, times');
+        end
+
+        function TimeUnitDefaultValue(tc)
+            % Verify that the default value of "TimeUnit" is "second".
```

Comment:
I think you forgot to check that the `array.Type.TimeUnit` is equal to `Second` based on the comment.